pkg/apiclients/testapi/testapi.go

```diff
@@ -231,6 +232,13 @@ func NewTestClient(serverBaseUrl string, options ...ConfigOption) (TestClient, e
 // Create the initial test and return a handle to poll it
 func (c *client) StartTest(ctx context.Context, params StartTestParams) (TestHandle, error) {
 	// Validate params
+	if params.LocalPolicy != nil && params.LocalPolicy.RiskScoreThreshold != nil {
+		threshold := *params.LocalPolicy.RiskScoreThreshold
+		if threshold > 1000 { // uint16 cannot be < 0
```

Comment:
Question: Is this a check we should do in the client application? This might make future changes more difficult. I would expect such checks in the backend with appropriate errors reported back.